### PR TITLE
commiting migrations

### PR DESCRIPTION
**Pull request by nohahesham**
@m0hamed  plz accept the ewo migrations

### CODE REVIEW

File: arability/config/environment.rb [hunk 1]
```diff
@@ -2,4 +2,4 @@
 require File.expand_path('../application', __FILE__)
 
 # Initialize the rails application
-Arability::Application.initialize!
+# Arability::Application.initialize!
```
Comment (m0hamed) [inline arability/config/environment.rb]: why is this line changed.I dont want anything other than the migration in the branch. I dont know what this line does but it doesnt look good

Comment (nohahesham) [inline arability/config/environment.rb]: i changed it back

**Comment by nohahesham** [comment]
@m0hamed
